### ISSUE TITLE
`vjs-big-play-centered` css class no longer centers play button in 5.0

### ISSUE THREAD

**Issue by meleyal** [videojs/video.js#2260]

**Comment by mmcc** [comment]
That's because I'm opinionated and I think centered play buttons are the bane of all well-meaning poster images. For too long have the faces of nice people in the poster cowered in the shadow of their centered play button oppressors!

![centered play buttons are bad and should feel bad](https://cloudup.com/cWM5Cglfze1+)

But seriously, although I don't like centered play buttons, I also don't want to spend the next year getting emails about new issues on this or bumps on this thread, so I'm not opposed to adding it back. Preferably everyone would start agreeing with me instead, but I'm not going to hold my breath :)

**Comment by heff** [comment]
> I also don't want to spend the next year getting emails about new issues on this or bumps on this thread

ditto

**Comment by meleyal** [comment]
This could be solved by redesigning the play button, e.g. flowplayer is fairly unobtrusive:
https://flowplayer.org/demos/

**Comment by mmcc** [comment]
I wouldn't consider it solved, since that's still a play button in the center of the interesting content, but I see what you mean.




I've got a PR ready for adding the centering class back, just fixing a few other minor 5 css issues while I'm there. 



—

**Comment by EZWrighter** [comment]
I must say, the play button is the action you want users to take so they can get to the video, the background/poster image is bait to get them to click it.  Moving that action button to a place that is not intuitive (Top left corner) is bad UI IMHO.  There is a reason the rest of the world centers their play buttons :-)  Maybe add an easy option to place it in the corners and leave it centered by default.

**Comment by mmcc** [comment]
> There is a reason the rest of the world centers their play buttons 

That's not really true, there are quite a few that don't center their play button. [iPlayer](http://www.bbc.co.uk/iplayer/episode/b0621g1v/the-syndicate-series-3-episode-6) is a great example (bottom left), and it's got a small audience. :) I was surprised to see that it seems *more* common in the UK than a centered play button, particularly among the major broadcasters.

Besides, just because there's something that current players do doesn't make it bad UI, it just makes it the status quo. I'm fine with having a class to allow people to center it, but I disagree that it should be the default. In fact, that class is [now back](2293), so this can be closed.

**Comment by meleyal** [comment]
Not to beat a dead horse but now that it's the status quo it is kind of bad UI because it confounds your expectations. If video.js is meant as a drop-in solution for video playback I don't think it's the right place for opinionated UI, and a centered play button should be the default.

**Comment by mmcc** [comment]
**TL;DR**: The play button isn't getting centered by default, but you are absolutely free to use the centered class if you'd like. The horse is dead and buried, so let the poor thing rest in peace.

The big play button is centered in some very popular players such as YouTube and other open source players. It isn't centered in some other *very popular* players. Both of these things are true. As I said in the comment before yours, if you were watching a broadcast television shows in the UK, for instance, it would be incredibly likely that the play button would **not** be centered. 

I'm sure there were people that complained about it at first, but then they got over it when they realized that the play button doesn't actually *do* anything. The entire dang poster is clickable and all the play button does is indicate "hey I'm a video, you should watch me." I would be shocked (and dismayed) if anyone that has used a modern browser has ever seen a player with a non-centered play button and been confounded as to what to do next.

![BBC: not centered](https://cloudup.com/ceBgDpMlP_J+)

![C4: also not centered](https://cloudup.com/cQSQOV2KH3D+)

Bask in the glory that is a poster image realizing its full potential. Those chefs are chopping the crap out of something and that guy is super angry(?) about some Minecraft. These might actually be the worst poster examples ever to illustrate my point, but I feel the same way about centered play buttons as that guy does about Minecraft (assuming he feels anger).

```
</soapbox>
```
